polaris-react/src/components/SkeletonPage/SkeletonPage.tsx

```diff
@@ -77,23 +77,18 @@ export function SkeletonPage({
       >
         <AlphaStack gap="0" fullWidth>
           <Box
-            padding={{xs: '4', md: '5'}}
-            paddingBlockEnd={{xs: '2', md: '5'}}
-            paddingInlineStart={{sm: '0'}}
-            paddingInlineEnd={{sm: '0'}}
+            paddingBlockStart={{xs: '4', md: '5'}}
+            paddingBlockEnd={{xs: '4', md: '5'}}
+            paddingInlineStart={{xs: '4', sm: '0'}}
+            paddingInlineEnd={{xs: '4', sm: '0'}}
           >
             {breadcrumbMarkup}
-            <Inline align="space-between" blockAlign="start">
+            <Inline align="space-between" blockAlign="center">
               {titleContent}
               {primaryActionMarkup}
             </Inline>
           </Box>
-          <Box
-            paddingBlockStart={{xs: '2', md: '5'}}
-            paddingBlockEnd="2"
-            paddingInlineStart="0"
-            paddingInlineEnd="0"
-          >
+          <Box paddingBlockEnd="2" paddingInlineStart="0" paddingInlineEnd="0">
```

Comment:
Do you need to zero out these values?

```suggestion
          <Box paddingBlockEnd="2">
```

Reply:
✅